indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -77,7 +78,20 @@ export default function FormItem({
             />
           ) : (
             <Form.Field required={isRequired || meta.alwaysRequired} styleName="field">
-              <label style={disabled ? {opacity: 0.8} : null}>{title}</label>
+              <label style={{opacity: disabled ? 0.8 : 1, display: 'inline-block'}}>{title}</label>
+              {setupMode && !!retentionPeriod && (
+                <Icon
+                  name="clock outline"
+                  color="red"
+                  style={{marginLeft: '3px'}}
+                  title={PluralTranslate.string(
+                    'Data retention period set to 1 week',
```

Comment:
Better to use the placeholder here as well. There are languages without plural forms, and IIRC it can cause some problems if the singular string doesn't contain the placeholder.

Reply:
The text sounds a bit like a message after changing it. How about sth. like "Field data will be purged X weeks after the event ended." instead?